images/ci-openshift-golang-builder-previous.yml

```diff
@@ -1,3 +1,4 @@
+mode: disabled
```

Comment:
The reference in line 18 "rhel-9-golang-1.19" will not exist, can we remove it, maybe replace with "rhel-9-golang"